instrumentation/kafka/kafka-connect-2.6/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/kafkaconnect/v2_6/KafkaConnectTask.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.kafkaconnect.v2_6;
+
+import java.util.Collection;
+import org.apache.kafka.connect.sink.SinkRecord;
+
+public final class KafkaConnectTask {
+
+  private final Collection<SinkRecord> records;
+
+  public KafkaConnectTask(Collection<SinkRecord> records) {
+    this.records = records;
+  }
+
+  public Collection<SinkRecord> getRecords() {
+    return records;
+  }
+
+  public static String getSpanName(KafkaConnectTask task) {
+    return "KafkaConnect.put";
```

Comment:
It would be better to use ` MessagingSpanNameExtractor.create(getter, operation)` for the span name since `KafkaConnect.put` doesn't conform to messaging span name conventions https://github.com/open-telemetry/semantic-conventions/blob/v1.24.0/docs/messaging/messaging-spans.md#span-name